_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,288 @@
+
+In this section, we will briefly go over the various steps we propose to take (Table 2) as we go about relicensing the
+[MDAnalysis core library][].
+
+### 1. A GPL v2+ package and source code (MDAnalysis v2.5 and lower)
+
+This is the original state of the license up until the [2.5.x releases of MDAnalysis](https://www.mdanalysis.org/blog/page2/#releases-2-5-0-and-2-4-x-of-mdanalysis).
+The package and source code were released under a [GPL v2+][GPL v2] license.
+
+### 2. A GPL v3+ package, and a mix of historical GPL v2+ and new LGPL v2.1+ source code (MDAnalysis v2.6 onwards)
+
+As of commit [44733fc][], all new contributions to MDAnalysis have been made under the terms of the
+[LGPL v2.1+][LGPL v2.1] and the package is released under [GPL v3+][GPL v3] to better reflect our
+understanding of the [Apache 2.0 incompatibility issues](#why-the-interim-gpl-v3-and-lgpl-v3-package-licenses-the-issue-with-apache-20).
+Historical source code contributions remain under the terms of [GPL v2+][GPL v2].
+
+This will remain the state of MDAnalysis until we obtain approval from our developers to relicense
+their code to [LGPL v2.1+][LGPL v2.1]. As we obtain contributor agreements, these historical
+source code contributions will slowly turn to [LGPL v2.1+][LGPL v2.1].
+
+### 3. A LGPL v3+ package, and an LGPL v2.1+ source code (Optional)
+
+Should we obtain approval to relicense from our developers before we remove our [Apache 2.0][]
+dependencies, we will then start releasing the MDAnalysis package under a [LGPL v3+][LGPL v3] license.
+
+### 4. An LGPL v2.1+ package and source code
+
+This is the state which we aim the library to eventually be licensed under. Once we both obtain
+the agreement of our developers to relicense and remove [Apache 2.0][] dependencies, we will be
+able to release the MDAnalysis package under [LGPL v2.1+][].
+
+## How does this affect me?
+
+### As a contributor
+
+If you have any historical contributions you will be asked to relicense them to [LGPL v2.1+][LGPL v2.1]
+instead of the initially proposed [LGPL v3+][LGPL v3].
+
+### As a developer
+
+Ultimately we believe that the proposed change to [LGPL v2.1+][LGPL v2.1] should offer a better compatibility
+(Table 1) with whatever license you may choose to use for your own code. The interim package license changes
+may however impact you, if you believe this to be the case [please do get in touch with us](#getting-in-touch).
+
+### As a user
+
+We do not anticipate this to have a significant impact on users. There may be some slight changes to how you
+[might access MMTF files which are discussed below](#removing-apache-20-dependencies).
+
+## Next steps
+
+### Contacting contributors
```

Comment:
What if the contributor is not the copyright owner? Should we provide a more general statement that the contributor confirms that the copyright owner agrees to the change? (I'm thinking of cases where someone contributes but the copyright is technically with the employer).

Reply:
For examples, this is what was written in Spack:

> By granting this permission, you are confirming that you have the right to make your contribution(s) to Spack and to grant this permission, and that if you made your contribution(s) in the course of your employment, that such contributions and consent have been authorized by your employer.

Reply:
See https://github.com/spack/spack/issues/9137

Reply:
The wording we have is lawyer approved already - we're not changing from it.

The question on being the copyright holder is right to ask but also very complicated legally and not something I would address here - I'll add a comment about asking us if they have any concerns about it.

At the end of the day this is lawyer realm. and we might have to go back to NF with questions.

Reply:
Well, it think this might be a problem for people that contributed while working for an employer that holds the copyright. The phrasing we have says specifically "I am the copyright owner", and does not account for cases where the contributor isn't (even if he has the consent of their employer for the relicensing). (Obligatory disclaimer: I'm not a lawyer.)

Reply:
All I can say here is "we know" - but this is lawyers orders.

Reply:
See what I sent you on discord :)

Reply:
I've added a note about being the copyright onwer - as per the intended text of the full email, we'll just ask folks to let us know if they don't know and we will just have to workshop it with them. My hope is that for our major contributors we can get blanket exceptions from say Manchester, Oxford, ASU, etc..

Reply:
Blanket exceptions would be great. I think the note looks good.